package.json

```diff
@@ -153,21 +153,21 @@
     "moment": "2.30.1",
     "moment-timezone": "0.5.46",
     "omit-empty-es": "1.2.0",
-    "patch-package": "6.5.1",
+    "patch-package": "^8.0.0",
```

Comment:
This is a breaking change upgrade. 
Since tests are already passing it seems it's safe but I wanted to know if we checked the breaking changes in the new two major versions.

Reply:
the breaking changes were all related to updating minimum node versions to those we have already moved past, and removal of deprecated methods we never used